docs/topics/testing/advanced.txt

```diff
@@ -568,6 +568,13 @@ Methods
 
     Returns the result produced by the running the test suite.
 
+.. method:: DiscoverRunner.get_test_runner_kwargs()
+
```

Comment:
Add `.. versionadded: 1.11` and a mention in docs/releases/1.11.txt.  (See our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).)